doc/modules/clustering.rst

```diff
@@ -798,6 +819,7 @@ by black points below.
 .. |dbscan_results| image:: ../auto_examples/cluster/images/sphx_glr_plot_dbscan_001.png
         :target: ../auto_examples/cluster/plot_dbscan.html
         :scale: 50
+        :alt:
```

Comment:
:alt: A bubble chart that maps three clusters, with the x-axis from -2.5 to 2 and the y-axis from -2 to 2.5. Outliers are indicated separately and are evenly spaced around the clusters.